Symfony/CS/Fixer.php

```diff
@@ -134,7 +134,7 @@ public function fix(ConfigInterface $config, $dryRun = false, $diff = false)
             $this->stopwatch->openSection();
         }
 
-        $fileCacheManager = new FileCacheManager($config->usingCache(), $config->getDir(), $config->getFixers());
+        $fileCacheManager = new FileCacheManager($config->usingCache(), $config->getCacheDir(), $config->getFixers());
```

Comment:
So, with this change you assume that someone has a config with getCacheDir, and on 1.6 he may not have it due to lack of that method in `ConfigInterface`. Therefore it is a BC break and should be targetted 2.0. Then you can cherry-pick #1076 into this PR.

About `Config::usingCache` part... yes, it should be merged into 2.0 too, our bad.